tasks/shared_libraries/recovery/shared_recovery.py

```diff
@@ -14,9 +14,8 @@ class RequestMethod(Enum):
     An enumeration.
     Provides potential actions for the database lambda to take when posting to the SQS queue.
     """
-
-    NEW = "post"
-    UPDATE = "put"
+    NEW_JOB = "new_job"
```

Comment:
If changing from NEW to NEW_JOB and UPDATE to UPDATE_FILE you will need to let Rizbi know as that will affect his code.